wp-content/themes/wporg-learn/template-parts/component-video-grid.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Template part for displaying a list of videos in a grid.
+ *
+ * @link https://codex.wordpress.org/Template_Hierarchy
+ *
+ * @package WPBBP
+ */
+
+$args = array(
+	'post_type' => 'wporg_workshop',
+	'category_name' => 'featured',
```

Comment:
The label for the section is "Recent" workshops, but this is specifically grabbing ones in the `featured` category. Once there are a lot of workshops, it'll definitely make sense to curate these a bit with a category, but for now, it might be better to just grab the most recently published ones.